security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.12.0"]
+== Release notes for {cert-manager-operator} 1.12.0
+
+Issued: 2023-10-02
+
+The following advisory is available for the {cert-manager-operator} 1.12.0:
+
+* link:https://access.redhat.com/errata/RHEA-2023:5339[RHEA-2023:5339]
+
+[id="cert-manager-operator-1.12.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, the {cert-manager-operator} did not allow you to configure the CPU and momeory limit for the cert-manager components and {cert-manager-operator}. Now, you can configure the CPU and memory limit by using the command-line interface (CLI). For more information on how to override CPU and memory limits for the cert-manager components and {cert-manager-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-13830[*OCPBUGS-13830*])
+
+* Previously, if you updated the `ClusterIssuer` object, the {cert-manager-operator} could not verify and update the change in the cluster issuer. Now, if you modify the `ClusterIssuer` object, the {cert-manager-operator} verifies the ACME account registration and updates the change. (link:https://issues.redhat.com/browse/OCPBUGS-8210[*OCPBUGS-8210*])
+
+* Previously, the {cert-manager-operator} did not support enabling the  `--enable-certificate-owner-ref` flag. Now, the {cert-manager-operator} supports enabling the `--enable-certificate-owner-ref` flag by adding the `spec.controllerConfig.overrideArgs` field in the `cluster` object. (link:https://issues.redhat.com/browse/CM-98[*CM-98*])
```

Comment:
Please consider adding another sentence in the note that,
`Enabling this flag lets cert-manager automatically deletes secret when the Certificate resource is removed from the cluster`

Reply:
Addressed